packages/easy-coding-standard/build/target-repository/.github/workflows/test.yaml

```diff
@@ -0,0 +1,23 @@
+name: Fixer Test
+
+on:
+    pull_request: null
+    push:
+        branches:
+            - main
+
+jobs:
+    fixer_test:
+        runs-on: ubuntu-latest
+
+        steps:
+            -   uses: actions/checkout@v2
+
+            -   uses: shivammathur/setup-php@v2
+                with:
+                    php-version: '7.3'
+                    coverage: none
+
+            -   uses: "ramsey/composer-install@v1"
+
```

Comment:
Here must be the install steps. Basically the same you've written manually in "how to reproduce" coment.